skimage/filters/tests/test_thresholding.py

```diff
@@ -563,6 +564,9 @@ def test_multiotsu_output():
     thresholds = [64, 128]
     assert np.array_equal(thresholds, threshold_multiotsu(image))
 
+    image = color.rgb2gray(data.astronaut())
+    assert 0.438 < threshold_multiotsu(image, 2) < 0.440
```

Comment:
Can you try to use assert_almost_equal please?

Reply:
Thank you @sciunto for your review. I will come soon with a new PR with a complete rewrite of `threshold_multiotsu` fixing the different issues. It is then may be not necessary to merge this PR.

Reply:
We are on the way to release 0.16. To this end, a quick fix might be better than a complete rewrite that will take time to review. ;) Cc @jni.